Update release notes page titles to include product name (#548)
AdobeDocs/commerce-extensibility: src/pages/admin-ui-sdk/release-notes.md

### Enhancements

> * This release changes how extensions are integrated into the Adobe Commerce Admin. Previously, the Admin directly loaded published extensions. Now, extensions must be selected in the **Configure Extensions** section of the Admin UI SDK configuration page.

Author: ```suggestion
* This release changes how extensions are integrated into the Adobe Commerce Admin. Previously, the Admin directly loaded published extensions. Now, extensions must be selected in the **Configure extensions** screen, resulting in a better developer experience. The limitation on **Production**-only workspaces has been resolved.
```

### Enhancements

> * This release changes how registrations are declared. Previously, calls were performed asynchronously. This could result with delayed retrieval and availability of registrations when loading pages. Registrations are now declared synchronously.

Author: ```suggestion
* This release changes how registrations are declared. Previously, calls were performed asynchronously. This could result in delayed retrieval and availability of registrations when loading pages. Registrations are now declared synchronously.
```

>    You must update any extension points developed with Admin UI SDK 1.x. [Migrate your extension point from 1.x to 2.0](./extension-points/index.md#migrate-your-extension-point-from-version-1x-to-version-20).

Author: ```suggestion
   You must update any extension points developed with Admin UI SDK 1.x. For details, see [Migrate your extension point from 1.x to 2.0](./extension-points/index.md#migrate-your-extension-point-from-version-1x-to-version-20).
```

* Added the **Refresh registrations** button to the configuration page. This button allows the administrator to  refresh the `admin_ui_sdk` cache and reload all extensions. <!--- CEXT-2642 -->

> * Refactored multiple sanitizers. As a result, incomplete or invalid input data from an extension point no longer causes display problems in the Admin.  <!--- CEXT-2649, CEXT-2732, CEXT-2731, CEXT-2728 -->

Author: ```suggestion
* Refactored multiple sanitizers. As a result, incomplete or invalid input data from an extension point no longer causes display problems in the Admin.  <!---  CEXT-2649, CEXT-2732, CEXT-2731, CEXT-2730, CEXT-2727, CEXT-2767 -->
```
